css/main.css

```diff
@@ -12,16 +12,24 @@ body {
   min-width: 450px;
```

Comment:
Override this for mobile since cause horizontal scroll. E.g. on 360px screen body with is still 450px.

Reply:
done